docking.js

```diff
@@ -1534,6 +1564,7 @@ var DockManager = class DashToDock_DockManager {
 
         Me.imports.extension.dockManager = this;
 
+        this.desktopIconsUsableArea = new DesktopIconsIntegration.DesktopIconsUsableAreaClass();
```

Comment:
```suggestion
        this._desktopIconsUsableArea = new DesktopIconsIntegration.DesktopIconsUsableAreaClass();
```

I'm wondering if it would be relevant to add this on request in case of intellihide being enabled. So in case `this._desktopIconsUsableArea` is defined we can update it otherwise we ignore the changes.

Reply:
The point is that this object is accessed from outside, from the DashToDock class (line 1115 and nexts), so it should be "public".

Reply:
Ok that's fine, but in such cases I'd prefer to have a `get desktopIconsUsableArea()`, so that it's clear that can't be set outside. Just to follow the same we're doing with other objects dockManager exposes.

Reply:
Good point. One second...

Reply:
Done.